### ISSUE TITLE
V2 A11y Considerations

### ISSUE THREAD

**Issue by kevinandrews1** [bbertucc/equalify#515]
Call info recording with @bbertucc:
- https://us06web.zoom.us/rec/share/FpIXSQ-fWq8q1wrmtDfqxqDYbIx1u7v8EFL9372VsvywqLtCj2vbLgW2fl4rkw9s.72ZbyLKBXIGUIcnj
- Passcode: 92swi4+H

And here are my notes:
- [ ] Didn't hear any notification that login was successful.
- [x] Should be clearer how to enter the URLs (maybe have a message and use aria-describedby) as some kind of helper text.
- [ ] "2 pages added" message not communicated to screen readers.
- [x] Was a little confusing; seems like step 1 has multiple steps even after you add the URL to the audit—like step 1 is multistep.
- [ ] More content on the page after the URLs added. Announce "2 pages added," click to next step; the focus area can shift to the next step since at that point it also seems like the next step.
- [x] Have "Added pages" navigation with the ability to remove pages. Add a dropdown to choose how many pages to show (e.g., if there are 300 pages to show).
- [x] For the table of checks, have the check name as the label for each checkbox.
- [x] Add more context to the checks under "select checks" or step 2, describing what this is.
- [ ] "2 checks added" wasn't announced to screen readers.
- [x] On step 3, skip h1 to h3; should be h2.
- [ ] The message after the audit is created doesn't get announced to screen readers (e.g., the audit ran and the number of issues found and pages, etc.).
- [x] Hide/show view options instead of showing them by default.
- [ ] After performing step 8, screen readers should announce "One item ignored in Lucy's Audit."
- [ ] After performing step 9, screen readers should announce "Ticket generated with two items from Lucy's Audit."
- [x] Step 13: I attempted to activate "Log 3" link, but nothing seemed to happen. It seemed to be stuck since at step 14, it also didn't do anything, so I closed out and started over.
- [x] The second time I did this, on step 3, when naming the audit, the screenshot checkbox was available to me when it wasn't available the first time.
- [ ] On step 9 the second time, I found the copy and close buttons. Pressing close doesn't seem to actually close the dialog.
- [x] When I attempted steps 13 and 14 the second time, they didn't seem to lead anywhere.
- [x] It seems I'm stuck at this point. I tried going to "Account," and it doesn't go anywhere.

**Comment by bbertucc** [comment]
Thanks @kevinandrews1 - I'll be getting to these 

Adding a bunch of issues from a call I did with Lucy:
- [x] <Fieldset> with <legend> - maybe replace with Aria grouping?
- [x] By default have view options closed
- [x] Make sure the Check all items checkbox isn’t a TH
- [x] Add label to each row checkbox
- [x] Add table summary to all tables
- [x] Change title to issue list: So it’s grouped by issue
- [x] Hide and show details doesn’t work on NVDA. There’s two targets for some reason. Maybe it’s because there’s a link - use has popup and 
- [x] Dialogue doesn’t move focus to it. Maybe aria hidden not in the dialogue
- [x] Dialogue isn’t showing in NVDA on Firefox
- [x] Nested lists read as silence in NVDA
- [x] Update New User Email address 
- [x] Instead of saying page items just say (10)

**Comment by bbertucc** [comment]
@kevinandrews1 I updated everything. A few notes:

- Unfortunately, since this is just raw HTML mockups, Jaws seems to ignore the assertive alerts. You can tab below H1 elements to see what it will read out once we move from mockups to the production-ready version, alerts will be read
- Focus area also can't shift in the HTML mockups, but I left your comments unchecked so we make sure to test it in the next version.

Can you go ahead and perform a second audit and report on issues? Here is the URL you'll use: https://equalifyeverything.github.io/equalify-v2-dashboard/

**Comment by bbertucc** [comment]
@kevinandrews1 we're moving super fast on this, so actually hold off on another validation until we finish the next phase of development.
